src/app/reducers/area.reducer.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { AreaReducer } from './area.reducer';
+import { initialState as AreaInitialState, AreaState } from './../states/area.state';
+import * as AreaActions from './../actions/area.actions';
+import { AreaUI } from './../models/area.model';
+export type Action = AreaActions.All;
+
+describe('AreaReducer', () => {
```

Comment:
Please change to `AreaReducer :`

Reply:
Fixed https://github.com/fabric8-ui/fabric8-planner/pull/2630/commits/1274d2632d41adc040909755d832b507fbd121e3